README.md

```diff
@@ -1,10 +1,10 @@
 # terraform-aws-ksoc-connect
 
-Allows KSOC to connect to your AWS account to perform pro-active monitoring of your AWS resources.
+Allows KSOC to connect to your AWS account to allow KSOC to be able to scan and anyalize your AWS resources.
```

Comment:
Don't think we need the double allow here.

Reply:
Removed the double allow